packages/react-ui/components/Kebab/Kebab.tsx

```diff
@@ -197,13 +218,21 @@ export class Kebab extends React.Component<KebabProps, KebabState> {
           this.setState({ focusedByTab: true });
         }
       });
+      this.props.onFocus?.();
```

Comment:
Когда фокус "восстанавливается", то коллбек срабатывает лишний раз. Поэтому надо добавить проверку, что кебаб "закрыт":
```suggestion
      !this.state.opened && this.props.onFocus?.();
```